### PR TITLE
Jyeah (fixes loadout items armour values) [(now)MODULAR]

### PR DESCRIPTION
**Pull request by Scuhf-git**
as the name implies

## About The Pull Request

Alternative for https://github.com/Bubberstation/Bubberstation/pull/734

## Why It's Good For The Game

Fixes the needless armour values that certain loadout items have so individuals cannot use them to power game.

## Changelog

:cl:
fix: Removed armour for Colourable Cloth Wraps
fix: Role restricts Tactical Boots
/:cl:

### CODE REVIEW

**Comment by Scuhf-git** [comment]
Tested obviously, it's itty bitty baby changes so like no issues.

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 1]
```diff
@@ -58,6 +58,7 @@
 	name = "Tactical Boots"
 	item_path = /obj/item/clothing/shoes/combat/rax
 	//ckeywhitelist = list("raxraus")
+	restricted_roles = list(JOB_BLUESHIELD, JOB_HEAD_OF_SECURITY, JOB_SECURITY_OFFICER, JOB_WARDEN, JOB_DETECTIVE, JOB_SECURITY_MEDIC, JOB_CORRECTIONS_OFFICER) //BUBBER EDIT
```
Comment (Majkl-J) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm:61]: This has been done modularly in #734

Either remove this edit or make it modular

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm [hunk 2]
```diff
@@ -88,7 +88,7 @@ GLOBAL_LIST_INIT(loadout_gloves, generate_loadout_items(/datum/loadout_item/glov
 
 /datum/loadout_item/gloves/armwraps
 	name = "Colourable Arm Wraps"
-	item_path = /obj/item/clothing/gloves/bracer/wraps
+	item_path = /obj/item/clothing/gloves/wraps //BUBBER EDIT (removed "bracer" path before "wraps")
```
Comment (Majkl-J) [inline modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm:91]: Can be modularised with an override

File: modular_skyrat/modules/customization/modules/clothing/hands/hands.dm [hunk 3]
```diff
@@ -31,7 +31,7 @@
 	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
 	resistance_flags = NONE
 
-/obj/item/clothing/gloves/bracer/wraps
+/obj/item/clothing/gloves/wraps  //BUBBER EDIT (removed "bracer" path before "wraps")
```
Comment (Majkl-J) [inline modular_skyrat/modules/customization/modules/clothing/hands/hands.dm:34]: Make your own type, or override the armor values, removing the skyrat one with no overrides might result in invalid type path issues whenever they choose to add these elsewhere in the code

File: modular_skyrat/modules/modular_vending/code/clothesmate.dm [hunk 4]
```diff
@@ -44,7 +44,7 @@
 				/obj/item/clothing/glasses/thin = 5,
 				/obj/item/clothing/glasses/hud/ar/projector = 5,
 				/obj/item/clothing/neck/face_scarf = 5,
-				/obj/item/clothing/gloves/bracer/wraps,
+				/obj/item/clothing/gloves/wraps, //BUBBER EDIT (removed "bracer" path before "wraps")
```
Comment (Majkl-J) [inline modular_skyrat/modules/modular_vending/code/clothesmate.dm:47]: If you just modularly override the armor values you don't have to do any of this

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm (hunk 1 above)
Comment (Majkl-J) [reply modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm:61]: Also add an "Alternative to #734" into the pr description to let maintainers know the close the other if this gets through

**Review by Majkl-J — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm (hunk 1 above)
Comment (Scuhf-git) [reply modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm:61]: Oh mb I didn't even know there was one done already i'll make it modular rq, and add that alternative comment.

File: modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm (hunk 2 above)
Comment (Majkl-J) [reply modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm:91]: Also doesnt have to be done if you make an override of the `/obj/item/clothing/gloves/bracer/wraps` armor values

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm (hunk 1 above)
Comment (Scuhf-git) [reply modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm:61]: Should be all good now, I did this at like 2 AM so that probably helps explain why it sucked so hard besides me naturally sucking hard at this.

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 5]
```diff
@@ -161,7 +162,6 @@
 	name = "Caligram Armored Tan Parka"
 	item_path = /obj/item/clothing/suit/armor/vest/caligram_parka_vest_tan
 //	ckeywhitelist = list("farsightednightlight", "raxraus", "1ceres", "marcoalbaredaa", "itzshift_yt", "drifter7371", "AvianAviator", "Katty Kat", "Investigator77", "Dalao Azure", "Socialistion", "ChillyLobster", "Sylvara", "AmZee", "Tf4", "rb303", "Kay_Nite", "whataboutism", "taac", "Halkyon", "Lupo_di_rosa", "Merek2", "lowpowermia", "RyeanBread", "Jesterz7", "Saund_Minah", "Ruediger4")
-	restricted_roles = list(JOB_BLUESHIELD, JOB_HEAD_OF_SECURITY, JOB_SECURITY_OFFICER, JOB_WARDEN, JOB_DETECTIVE, JOB_CORRECTIONS_OFFICER)
```
Comment (YakumoChen) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm:164]: Why did you remove the restricted roles from an actual armour vest?

File: modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_shoes.dm
Comment (YakumoChen) [inline modular_zubbers/code/modules/loadouts/loadout_items/loadout_datum_shoes.dm:1]: You forgot to include the file

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm (hunk 5 above)
Comment (Scuhf-git) [reply modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm:164]: Wait wtf I didn't even see this mb, I was about the close this because I didn't notice the other PR just fixes the same exact issues thanks for pointing this out though I didn't see this.
